### PR TITLE
feat: `is_clique()` and `is_independent_vertex_set()`

### PR DESCRIPTION
**Pull request by szhorvat**
One question is whether we should drop `is_complete()` from Python now that `g.is_clique()` (with no parameters) performs the same function, and dispatches to the same C function.

Perhaps just keep it for discoverability?

### CODE REVIEW

**Comment by szhorvat** [comment]
Resolved conflicts and rebased.

**Comment by ntamas** [comment]
> Perhaps just keep it for discoverability?

Yes.

File: src/_igraph/graphobject.c [hunk 1]
```diff
@@ -13728,6 +13799,22 @@ struct PyMethodDef igraphmodule_Graph_methods[] = {
    "@return: C{True} if it is complete, C{False} otherwise.\n"
    "@rtype: boolean"},
 
+  {"is_clique", (PyCFunction) igraphmodule_Graph_is_clique,
+   METH_VARARGS | METH_KEYWORDS,
+   "is_clique(vertices, directed=False)\n--\n\n"
```
Comment (ntamas) [inline src/_igraph/graphobject.c:13804]: `is_clique()` supports `vertices=None` and `vertices` itself is optional. Here it seems that `vertices` is mandatory. I don't mind either way, you can make it mandatory or you can leave it as-is (when it supports `None`), just make sure that the docs are consistent with the behaviour.

File: src/_igraph/graphobject.c [hunk 2]
```diff
@@ -13728,6 +13799,22 @@ struct PyMethodDef igraphmodule_Graph_methods[] = {
...
+   "Decides whether a set of vertices is a clique, i.e. a fully connected subgraph.\n\n"
+   "@param vertices: a list of vertex IDs\n"
+   "@param directed: whether to require mutual connections between vertex pairs\n"
+   "    in directed graphs.\n"
+   "@return: C{True} is the given vertex set is a clique, C{False} if not.\n"},
+
+  {"is_independent_vertex_set", (PyCFunction) igraphmodule_Graph_is_independent_vertex_set,
+   METH_VARARGS | METH_KEYWORDS,
+   "is_independent_vertex_set(vertices)\n--\n\n"
```
Comment (ntamas) [inline src/_igraph/graphobject.c:13813]: Same as above: `vertices=None` is supported now and it is optional. The docs should be consistent with the behaviour.

**Review by ntamas — CHANGES_REQUESTED** [review]
Feel free to merge after addressing the review comments.

File: src/_igraph/graphobject.c (hunk 2 above)
Comment (szhorvat) [reply src/_igraph/graphobject.c]: Fixed.

File: src/_igraph/graphobject.c (hunk 1 above)
Comment (szhorvat) [reply src/_igraph/graphobject.c]: Fixed.

**Comment by szhorvat** [comment]
@ntamas PR merged, I think we're good for a release.

**Pull request merged**